storybook/src/pages/amopis/DetailsPage/DetailsPage.tsx

```diff
@@ -0,0 +1,163 @@
+import {
+  AspectRatio,
+  Breadcrumb,
+  Column,
+  DescriptionList,
+  Grid,
+  Heading,
+  Image,
+  Link,
+  LinkList,
+  Paragraph,
+  Row,
+  Tabs,
+} from '@amsterdam/design-system-react'
+import { EditDocumentIcon, EmailIcon } from '@amsterdam/design-system-react-icons'
+import '../common/amopis.css'
+import { Submenu } from '../common/Submenu'
+import { menuLevel3 } from '../common/config'
+
+export const DetailsPage = () => (
+  <Grid paddingVertical="medium">
+    <Grid.Cell span="all">
+      <Breadcrumb>
```

Comment:
All your pages have become static without reusing other stories, which was kind of the idea.

Reply:
We don’t have many stories for application screens. Reusing stories is very nice, but it is not required.